app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss

```diff
@@ -0,0 +1,50 @@
+@charset "UTF-8";
```

Comment:
basically anything that we didn't write (e.g. a library from someone else) we try to keep under the vendor folder